pkg/log/zap/zap.go

```diff
@@ -39,6 +39,7 @@ type NewEncoderFunc func(...EncoderConfigOption) zapcore.Encoder
 // New returns a brand new Logger configured with Opts. It
 // uses KubeAwareEncoder which adds Type information and
 // Namespace/Name to the log.
+// Deprecated: use bootstrap.New as instead.
```

Comment:
What is the reason for moving things around? If anyhow possible, lets avoid this.

Reply:
> What is the reason for moving things around?

I think using `zap` as the package name might bring confusion to users because we actually do not use `zap.Logger` but `logr.Logger`. 

As the controller-runtime still stay with `0.x.x`, I think maybe it would be friendly to introduce breaking changes right now rather than bring breaking changes after the stable `1.x.x`. :heart: 


> If anyhow possible, lets avoid this. 

I would make it a try! :D